kayak_core/src/styles/mod.rs

```diff
@@ -138,22 +143,29 @@ define_styles! {
     pub struct Style {
         pub background_color : StyleProp<Color>,
         pub border_color: StyleProp<Color>,
-        pub border_radius: StyleProp<(f32, f32, f32, f32)>,
-        pub border: StyleProp<(f32, f32, f32, f32)>,
+        pub border_radius: StyleProp<Edge<f32>>,
+        pub border: StyleProp<Edge<f32>>,
         pub bottom: StyleProp<Units>,
         pub color: StyleProp<Color>,
+        /// The spacing between child widgets along the horizontal axis
+        pub col_gap: StyleProp<Units>,
```

Comment:
Should `col_gap` and `row_gap` be renamed to match Morphorm's naming (`col_between` and `row_between`)? I chose "gap" to be more consistent with CSS, but I'm okay with either honestly.

Reply:
I think we should stay consistent with morphorm, I'm not sure I've always done that but it might help with documentation and type sharing in the future. 🤔